test_crates/trait_default_impl_removed/old/src/lib.rs

```diff
@@ -0,0 +1,20 @@
+// Default trait method impl is removed, should be reported
+pub trait TraitA {
+    fn method_default_impl_removed(&self) {}
+}
+
+// Default trait method becomes removed completely, should not be reported as it becomes
+// missing instead
+pub trait TraitB {
+    fn method_becomes_removed(&self) {}
+}
+
+//Default trait method impl is removed, but its private, shouldnt be reported
+trait TraitC {
+    fn method_default_impl_removed(&self) {}
+}
+
+// Non-object safe trait has its default impl removed, should be reported
```

Comment:
This comment is a bit misleading. It makes it sound like the trait is _currently_ not object-safe, whereas in reality the trait _becomes_ non-object-safe as part of the change in `old -> new`.

Which scenario was the test meant to catch? Let's make sure the description and the actual test are in sync.